mozilla/foundation.mozilla.org
# Hero video and images

Closes #6675 

**Link to sample test pages**:
Test Page with Video Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Test Page with Image Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/
Test Page with Neither: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/


**Steps to Test**:

1. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/, video should load at the top and start playing immediately.
2. Video should be responsive, and play/pause button should work as expected.
3. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/, there should be a responsive image at the top of the page. 
4. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/, it should look like previous blog posts.


**NOTE:** Once PR is approved, we will need to add vimeo to the allowed media sources in heroku.
## Checklist

**Changes in Models:**
